docs/writings/web-apps/part-1.md

```diff
@@ -0,0 +1,199 @@
+Organizations can integrate third-party services and tools directly into their oCIS environment. This can include CRM systems, marketing automation tools, or custom data visualization tools, providing a seamless workflow for users.
+
+3. **Enhanced Security and Compliance**:\
+Custom extensions can help organizations adhere to specific security and compliance requirements by adding features like custom authentication mechanisms, data encryption tools, or compliance reporting modules.
+
+4. **Branding and Identity**:\
+By customizing the look and feel of the web applications, organizations can ensure their brand identity is consistently represented across their digital platforms. This can include custom themes, logos, and color schemes.
+
+5. **Innovative Features**:\
+Custom extensions allow organizations to experiment with new features and functionalities that are not available in the default setup. This can include AI-powered tools, advanced analytics, or unique collaboration features.
+
+The ability to provide custom extensions makes oCIS a powerful and flexible platform that can adapt to the evolving needs of any organization.
+
+It empowers administrators to craft solutions that are not only functional but also aligned with their strategic goals.
+
+---
+
+**Loading Extensions in oCIS**
+
+In oCIS, extensions can be loaded at both build time and runtime. Understanding the difference between these two methods is key to effectively managing and utilizing extensions.
+
+- **Build Time Extensions**:\
+These are integrated into the binary during the build process. They are part of the core system and cannot be altered without rebuilding the system. This ensures a stable and consistent environment where critical applications are always available.
+
+- **Run Time Extensions**:\
+These are loaded dynamically at runtime, providing greater flexibility. They can be placed in a designated directory and are automatically picked up by the system. This allows administrators to easily add, update, or remove extensions as needed, without the need for a system rebuild.
+
+---
+
+**How to Load Extensions**
+
+1. **Build Time Extensions**:
+   - Located in `<ocis_repo>/services/web/assets/apps`.
+   - Integrated into the system during the build process.
+   - These extensions are part of the binary and cannot be modified at runtime.
+
+2. **Run Time Extensions**:
+   - Stored in the directory specified by the `WEB_ASSET_APPS_PATH` environment variable.
+   - By default, this path is `$OCIS_BASE_DATA_PATH/web/apps`, but it can be customized.
+   - Run time extensions are automatically loaded from this directory, making it easy to add or remove extensions without rebuilding the system.
+
+The ability to load extensions at runtime is particularly powerful, as it allows for a high degree of customization and flexibility. Administrators can quickly respond to changing needs by adding new functionality or removing outdated extensions.
+
```

Comment:
Do you want to add a paragraph on the technology that is possible to use for extensions?